### PR TITLE
Added Codechef Problems

### PR DESCRIPTION
**Pull request by vismay2303**
Added Comments which were missing earlier.

### CODE REVIEW

File: CodeChef_problems/LAPD/solution.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    ct=0
+    for i in range(1,B+1):
+        temp=0
+        for j in range(1,A+1):
+            for k in range(1,C+1):
+                if(i*i<((j-1)*(k-1))): #checking for all values possible b^2<a*c
+                    temp+=1
+                    ct+=1 #keeping the count of the values
+                    #print(j,i,k)
+        print(temp)
```
Comment (Vishruth-S) [inline CodeChef_problems/LAPD/solution.py:16]: Remove `print(temp)` at line 16.
This causes unnecessary output resulting in incorrect answer

Comment (vismay2303) [reply CodeChef_problems/LAPD/solution.py:16]: Rmoved

**Comment by vismay2303** [comment]
Pls check and let me know

**Review by Vishruth-S — COMMENTED** [review]
Yeah now it looks good,
Could you please delete the file `GDSUb - Shortcut.lnk` since it is unecessary.

**Comment by vismay2303** [comment]
Done

**Review by Vishruth-S — APPROVED** [review]
Okay 👍 Looks great now 😃 Thanks for your contribution
(Also do star this repo if you haven't already 🙃 )

**Comment by vismay2303** [comment]
Thanks

**Pull request merged**